query-engine/driver-adapters/connector-test-kit-executor/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "engines": {
     "node": ">=16.13",
-    "pnpm": ">=8.6.6 <9"
+    "pnpm": "8.10.2"
```

Comment:
Looks a bit strict, before was better I think

Reply:
Hm, probably we need to do the same as https://github.com/prisma/prisma-engines/pull/4418 then but for pnpm?

Reply:
I think so too

Reply:
(Opened https://github.com/renovatebot/renovate/discussions/25604 to figure out if there is a better way to iterate on the renovate config)

Reply:
And here is a PR: https://github.com/prisma/prisma-engines/pull/4419